python/felis/db/sqltypes.py

```diff
@@ -115,10 +115,12 @@ def compile_double(type_: Any, compiler: Any, **kw: Any) -> str:
     POSTGRES: postgresql.VARCHAR,
 }
 
-text_map: _TypeMap = {
-    MYSQL: mysql.LONGTEXT,
+# text() method wants to pass length argument to the type, but mysql
+# LONGTEXT and postgres TEXT do not support it.
+text_map: dict[str, Any] = {
+    MYSQL: lambda length: mysql.LONGTEXT(),
```

Comment:
I do not like this solution, as it feels more like a workaround than a proper fix for length being required. We should instead make the `text` type itself unsized, since this flag essentially means that length is required, which we do not want for unbounded text fields.

Reply:
This is a workaround, but it was done according to current documented felis model. If the proper fix requires making `text` type different, it may need an RFC, which means extra delay for me. I will workaround it in dax_ppdb in the meantime.

Reply:
> it may need an RFC

I don't believe so since `text` being broken in the past meant no one was using it in their schema files. I believe we can change the `is_sized` flag safely.